app/components/atoms/Button/Button.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { composeStories } from "@storybook/react";
+import { render, screen } from "@testing-library/react";
+import * as Stories from "./Button.stories";
+
+const { Default, Medium, Large, Disabled, WithIcon } = composeStories(Stories);
+
+describe("Button Component", () => {
+  test("render all buttons variants", () => {
+    render(
+      <>
+        <Default />
+        <Medium />
+        <Large />
+        <Disabled />
+        <WithIcon />
+      </>,
+    );
+
+    const buttons = screen.getAllByRole("button");
+
+    buttons.forEach((button) => {
+      expect(button).toBeInTheDocument();
+    });
```

Comment:
aqui acho que podia usar o [test.each](https://jestjs.io/pt-BR/docs/api#array-of-arrays) do jest em

Reply:
Bem lembrado 😶‍🌫️